e2e_playwright/conftest.py

```diff
@@ -610,6 +610,9 @@ def compare(
 
 def wait_for_app_run(page: Page, wait_delay: int = 100):
     """Wait for the given page to finish running."""
+    # Add a little timeout to wait for eventual debounce timeouts used in some widgets.
+    page.wait_for_timeout(155)
```

Comment:
This timeout isn't really required to fix the specific flakiness but might be a good addition to lower some possible flakiness around debounce.